firebase-inappmessaging/src/main/java/com/google/firebase/inappmessaging/FirebaseInAppMessaging.java

```diff
@@ -109,6 +110,42 @@ public boolean isAutomaticDataCollectionEnabled() {
     return dataCollectionHelper.isAutomaticDataCollectionEnabled();
   }
 
+  /**
+   * Enable, disable or clear automatic data collection for Firebase In-App Messaging.
+   *
+   * <p>When enabled, generates a registration token on app startup if there is no valid one and
+   * generates a new token when it is deleted (which prevents {@link
+   * com.google.firebase.iid.FirebaseInstanceId#deleteInstanceId} from stopping the periodic sending
+   * of data). This setting is persisted across app restarts and overrides the setting specified in
+   * your manifest.
+   *
+   * <p>When null, the enablement of the auto-initialization depends on the manifest and then on the
+   * global enablement setting in this order. If none of these settings are present then it is
+   * enabled by default.
+   *
+   * <p>If you need to change the default, (for example, because you want to prompt the user before
+   * generates/refreshes a registration token on app startup), add to your application’s manifest:
```

Comment:
@VinayGuthal 
Not sure what's intended here, so perhaps "...a registration token is generated/refreshed on app startup...."

Reply:
hmm I just copied this from the existing documentation for the older function

Reply:
@JasonAHeron what do you think?

Reply:
This seems clear to me, what are you hoping to change with the docs?

Reply:
I was just confirming if this was okay!